sql/telemetry_derived/scalar_percentiles_v1/query.sql

```diff
@@ -0,0 +1,346 @@
+CREATE TEMP FUNCTION udf_zip_arrays(keys ARRAY<FLOAT64>, values ARRAY<FLOAT64>)
+RETURNS ARRAY<STRUCT<key STRING, value FLOAT64>> AS (
+  -- Zips two arrays into a MAP type.
+  (
+    SELECT ARRAY_AGG(record)
+    FROM (
+      SELECT
+        STRUCT<key STRING, value FLOAT64>(CAST(k AS STRING), values[OFFSET(off)]) as record
+      FROM
+        UNNEST(keys) AS k WITH OFFSET off
+    )
+  )
+);
+
+WITH flat_clients_scalar_aggregates AS (
+  SELECT * EXCEPT(scalar_aggregates)
+  FROM clients_scalar_aggregates_v1
+  CROSS JOIN UNNEST(scalar_aggregates))
+  
+SELECT
+  os,
+  app_version,
+  app_build_id,
+  channel,
+  metric,
+  metric_type,
+  key,
+  agg_type AS client_agg_type,
+  'percentiles' AS agg_type,
+  -1 AS total_users,
```

Comment:
`total_users` for the set of dimensions there is already computed in `clients_scalar_probe_counts` so we don't need to compute it again. But that field is still here because the idea is to append this data with percentiles to the `client_probe_counts` table. With `percentiles` listed as the `agg_type`.

Could perhaps keep the percentiles table separate if this is too confusing...

Reply:
It is a bit confusing. Could we not calculate `total_users` correctly?

Reply:
We could. I've added it, I thought it would be redundant to have it in two places but I guess the lack of clarity is worse.